packages/core/src/index.js

```diff
@@ -82,26 +83,30 @@ class Editor {
     middleware = [],
     editables = [],
     defaultPlugin = pluginDefault,
-    dragDropBackend
+    dragDropBackend,
+    store
   }: {
     plugins: { content: [], layout: [], native?: any },
     middleware: [],
     editables: EditableType[],
     defaultPlugin: any,
-    dragDropBackend: any
+    dragDropBackend: any,
+    store: Store
   } = {}) {
     if (instance) {
       console.warn(
         'You defined multiple instances of the Editor class, this can cause problems.'
       )
     }
-
     instance = this
-    this.store = createStore(initialState(), middleware)
+
+    this.store = store || createStore(initialState(), middleware)
+    // If you use your own store, the store will be combined under the key 'ory'
+    this.getState = store ? () => this.store.getState().ory : this.store.getState
```

Comment:
I'm not sure if this is a good idea, it could mean that many selectors down the road (e.g. for cells) could break. I'm not sure if it does, but it might very well be the case. What's the downside of just merging the two states?

Reply:
Yeah this is main thing I wanted feedback on. Without this the end user would have to have 
```
  editables,
  display,
  focus,
  settings
```
as top level redux-store keys in their store. If there was only one such key and it had a sensible name this would be alright. Alternatively a compromise could be adding an `ory_` prefix to each of these keys. (Or nesting all these keys within an `ory` key - potentially a better solution). Having those keys as top level `store` `keys` is not a particularly attractive option.

However I may be missing a better way to compose reducers. I agree that my PR solution is not ideal in terms of future selectors.

Reply:
Yeah I see how that makes sense with an existing store. I guess encapsulating this in a subkey is the way to go then, but it *may* won't work very well with existing selectors. Have you tried if everything still works as it should? So moving cells, editing content, .... when a store is set?